plugins/org.eclipse.fordiac.ide.gef/plugin.properties

```diff
@@ -1,19 +1,20 @@
- ###############################################################################
- # Copyright (c) 2008, 2009, 2011, 2014 - 2016 Profactor GbmH, fortiss GmbH
- #				 2020 Andrea Zoitl
- # 
- # This program and the accompanying materials are made available under the
- # terms of the Eclipse Public License 2.0 which is available at
- # http://www.eclipse.org/legal/epl-2.0.
- #
- # SPDX-License-Identifier: EPL-2.0
- #
- # Contributors:
- #   Gerhard Ebenhofer, Alois Zoitl
- #     - initial API and implementation and/or initial documentation
- #   Andrea Zoitl
- #	   - externalized translatable strings
- ###############################################################################
+###############################################################################
+# Copyright (c) 2008, 2024 Profactor GbmH, fortiss GmbH, Andrea Zoitl,
+						   Primetals Technologies Austria GmbH
```

Comment:
Please add the missing `#` at the start of the line.

Reply:
fixed